recipes-graphics/libwebrtc/libwebrtc_125.6422.06.bb

```diff
@@ -35,7 +35,7 @@ SRCREV_libwebrtc = "a6522062f83cac380a4544e036114015cf061ffa"
 S = "${WORKDIR}/src"
 B = "${WORKDIR}/src/out/Linux-${GN_TARGET_ARCH_NAME}"
 
-inherit gn-fetcher features_check pkgconfig
+inherit gn-fetcher pkgconfig
```

Comment:
it can be as separate PR as well, because as I guess feature_check is not used at all